src/main/java/cn/nukkit/block/Block.java

```diff
@@ -130,6 +130,10 @@ public boolean onBreak(Item item) {
         return this.getLevel().setBlock(this, Block.get(AIR), true, true);
     }
 
+    public boolean onBreak(Item item, Player player) {
+        return onBreak(item);
+    }
+
```

Comment:
I'm going to add a clarification for anyone reviewing it.

It was added to allow special break behaviour based on the player who is breaking the block as seen later in this commit.